cleanup/azure.sh

```diff
@@ -63,4 +63,70 @@ else
   echo "== No dangling instance found to terminate."
 fi
 
+## Delete staging shared gallery images versions
```

Comment:
I'm wondering if these dev/staging image versions deletions should be done even on PR, and not as dry-run, WDYT?

Reply:
In that case I would put these commands in a separate script.